docs/quick_start.md

```diff
@@ -11,7 +13,10 @@
   * [Custom Deployment with Specific ClickHouse Configuration](#custom-deployment-with-specific-clickhouse-configuration)
 
 # Prerequisites
-1. Operational Kubernetes instance
+
+1. Kubernetes instance with the following version considerations:
+    1. `clickhouse-operator` versions **before** `0.16.0` requires [Kubernetes prior to `1.16`](https://kubernetes.io/releases/).
```

Comment:
compatible with kubernetes version prior 1.22

Reply:
Updated to 1.22.